keras_retinanet/bin/debug.py

```diff
@@ -201,9 +201,21 @@ def run(generator, args, anchor_params):
                 # result is that annotations without anchors are red, with anchors are green
                 draw_boxes(image, annotations['bboxes'][max_indices[positive_indices], :], (0, 255, 0))
 
-        cv2.imshow('Image', image)
-        if cv2.waitKey() == ord('q'):
-            return False
+        key = cv2.waitKey(1)
```

Comment:
You can still do `cv2.waitKey()`, so that it will wait indefinitely, right? That eliminates the need of the while loop below.

Reply:
Yup, I changed this to a single cv2.waitKey(1) and removed the while loop, code is a lot neater now.

Reply:
I meant `cv2.waitKey()`, without any arguments. If you don't pass any arguments it will just wait idly until a key is pressed, which in our case is sufficient.

Reply:
Alright, just made the change